database/firebird/firebird.go

```diff
@@ -0,0 +1,243 @@
+// +build go1.9
+
+package firebird
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/hashicorp/go-multierror"
+	_ "github.com/nakagami/firebirdsql"
+	"io"
+	"io/ioutil"
+	nurl "net/url"
+	"strings"
+)
+
+func init() {
+	db := Firebird{}
+	database.Register("firebird", &db)
+	database.Register("firebirdsql", &db)
+}
+
+var DefaultMigrationsTable = "SCHEMA_MIGRATIONS"
```

Comment:
Any reason why the schema migrations table name is all upper case?
Every other DB driver uses lower case names

Reply:
Fixed